Quoteau/Quoteau/Extensions/UIImage.swift

```diff
@@ -0,0 +1,66 @@
+//
+//  UIImage.swift
+//  Quoteau
+//
+//  Created by Wiktor Górka on 19/12/2019.
+//  Copyright © 2019 Lunar Logic. All rights reserved.
+//
+
+import UIKit
+
+extension UIImage {
+
+    func fixOrientation() -> UIImage? {
```

Comment:
Name 'fix orientation' doesn't really say much. Why was the orientation broken? What does it change? I'd also probably divide this function into smaller, namable, chunks, so that reading this code is a bit easier.